Project-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Footer.css';
+
+const Footer = () => {
+    return (
+        <div>
+            <img src="" alt="" />
+            <img src="" alt="" />
+            <img src="" alt="" />
+            <img src="" alt="" />
+        </div>
```

Comment:
it looks like you were attempting to use images for you icons in your footer section. you can also use font awesome for icons.